### ISSUE TITLE
Having issues with upgrade to Django 2.0

### ISSUE THREAD

**Issue by harelw** [erdem/django-map-widgets#44]
Might be an issue with how I am loading static files. This is the ouput from the JavaScript console when trying to render `GooglePointFieldWidget`:

`jquery_class.js:72 Uncaught ReferenceError: jQuery is not defined
    at jquery_class.js:72
django_mw_base.js:202 Uncaught ReferenceError: jQuery is not defined
    at django_mw_base.js:202
mw_google_point_field.js:102 Uncaught ReferenceError: jQuery is not defined
    at mw_google_point_field.js:102
127.0.0.1/:308 Uncaught ReferenceError: DjangoGooglePointFieldWidget is not defined
    at 127.0.0.1/:308
    at 127.0.0.1/:312`

**Comment by MUDASSARHASHMI** [comment]
Having same issue with 2.0. Its not issue with static because JS loading from other files withing MapWidget folder but these four files won't.

**Comment by Marsnk** [comment]
Place the {{ form.media }} template tag above the form that renders the map widget. In other words, instead of following the install instructions to place it in the HEAD or BODY:

```
# some_form.html
<div ...>
{{ form.media }}
<form method="POST" ...>
# form rendered manually or as_p that contains map widget
...
</form> 
</div>
```

See Axil's answer: https://stackoverflow.com/questions/47883230/django-map-widget-console-showing-djangogooglepointfieldwidget-uncaught-referenc

**Comment by harelw** [comment]
Shouldn't this work out of the box with django admin?

**Comment by dadokkio** [comment]
I'm using this plugin with django 2.0 (new project) and I confirm that it works out of the box with django admin. I just had to use a release newer than the one installed from pip to make it working also in standard form [with the {{form.media}} in the header]

Have you performed the collectstatic command?

**Comment by harelw** [comment]
Hmm seems like the old way of returning custom forms in admin causes issues:

```
    def get_form(self, request, obj=None, **kwargs):
         return VenueAdminAddForm
```
The following fixed it:

```
    def get_form(self, request, obj=None, **kwargs):
        kwargs['form'] = VenueAdminAddForm
        return super().get_form(request, obj, **kwargs)
```

**Comment by erdem** [comment]
Hey guys

Thank you for your contributions and feedbacks. 

As you mentioned above, the admin jQuery file doesn't load before mapwidgets static files. Also, there were a few Python 3 compatibility issues as I noticed before. However, I updated `v0.2.0` branch which is not released yet. It's working with Django 2.0. You can install the prerelease version with `pip` from the branch.  

`pip install git+git://github.com/erdem/django-map-widgets.git@v0.2.0`

Btw, I added an extra jQuery file to the widget static files. Before that, it was working with django admin jQuery. 

I will release the new version soon. 

Thanks

**Comment by amankumarjain** [comment]
Hi @erdem , when are you planning to release 0.2.0 version?

**Comment by erdem** [comment]
Hi @amankumarjain 

Unfortunately, I didn't spare time for this project so long. I need to focus my current job right now. I can't give you a precise date, but I will try to release ASAP. 

Until the new release, your pull-requests are welcome.

**Comment by erdem** [comment]
Hi All

I released version `0.2.0`, it's on the PyPI now. 

Thanks!
